tests/sensors/test_python.py

```diff
@@ -85,7 +85,7 @@ def test_python_callable_arguments_are_templatized(self):
 
         ds_templated = DEFAULT_DATE.date().isoformat()
         # 2 calls: first: at start, second: before timeout
-        assert 2 == len(recorded_calls)
+        assert 1 <= len(recorded_calls)
```

Comment:
This seems to defeat the purpose of this assertion. Why is the second call not recorded?

Reply:
Why do we have the second call at all? Can we not have it?

Reply:
It' sensor and number of calls depends on how fast the whole system is at all. We know tht the first call happens for sure, and most of the time the second too. The assert is actually correct IMHO ("expect at least 1 call").  We only check the first call anyway (and this is the gist of it as we really want to see if JINJA template works in the sensor call). 

If you think on how to fix it better I am al ears :)

Reply:
The thing here is that his is the "timing" tests  and we test it with "run" method and expect timeout. 

In perfect world, we could likely use freezegun and simulate the exact time passing, but IMHO it is totally not worth it if we can 100% fix it with this one character change. Trading test simplicity with perfectness.

Reply:
Works for me :)

Reply:
I also removed it from the test below (same story but it had a comment about it and was already using >= 1).